.github/workflows/check_transport.yml

```diff
@@ -19,6 +19,11 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      - name: Update /etc/hosts file
+        if: matrix.os == 'ubuntu-20.04'
```

Comment:
since we've determined that 18.04 is also affected, might as well use `startsWith(matrix.os, 'ubuntu-') wdyt?
source for syntax: https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#startswith

Reply:
done